beta/index.markdown

```diff
@@ -0,0 +1,183 @@
+---
+layout: page
+title:  "Beta"
+show_header_link: true
+categories:
+  - bonnyci
+tags:
+  - zuul
+  - ci
+  - testing
+  - gating
+---
+
+Avast! Welcome, and thanks for your interest in BonnyCI. It's about time
+we publicly explain what BonnyCI is, and why it works the way it does.
+
+
+The story begins with [OpenStack](http://www.openstack.org/). Early
+on in OpenStack's history, some of the
+developers identified a need for robust [Continuous
+Integration](https://en.wikipedia.org/wiki/Continuous_integration).
+This included testing multiple projects together to ensure
+they interact well. Developers who set this system up chose
+[Jenkins](http://jenkins.io/), and also decided to pursue a path of gated
```

Comment:
I love the reference links elsewhere in the file. Should we make Jenkins one too?